contracts/transfer/src/wasm/internal.rs

```diff
@@ -4,74 +4,63 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use crate::{PublicKeyBytes, TransferContract};
-use core::convert::TryFrom;
+use crate::{Error, Map, PublicKeyBytes, TransferContract};
 
 use alloc::vec::Vec;
-use canonical::{InvalidEncoding, Store};
+use dusk_abi::ContractId;
 use dusk_bls12_381::BlsScalar;
 use dusk_bytes::Serializable;
 use dusk_jubjub::JubJubAffine;
-use dusk_kelvin_map::Map;
-use dusk_pki::PublicKey;
+use dusk_pki::{PublicKey, PublicSpendKey};
 use phoenix_core::{Crossover, Fee, Message, Note};
-use rusk_abi::PublicInput;
+use rusk_abi::{PaymentInfo, PublicInput};
 
-impl<S: Store> TransferContract<S> {
-    pub(crate) fn push_fee_crossover(
-        &mut self,
-        fee: Fee,
-    ) -> Result<(), S::Error> {
+impl TransferContract {
+    pub(crate) fn push_fee_crossover(&mut self, fee: Fee) -> Result<(), Error> {
         let block_height = dusk_abi::block_height();
 
         // FIXME Get gas consumed
         // https://github.com/dusk-network/rusk/issues/195
         let gas_consumed = 2;
```

Comment:
Isn't the host function avaliable in this abi version?

Reply:
Probably yes. What is the current ABI call?

Reply:
Good catch, fixed